conans/client/cmd/user.py

```diff
@@ -33,10 +33,10 @@ def user_set(localdb_file, user, remote_name=None):
 
     if user.lower() == "none":
         user = None
-    return update_localdb(localdb, user, None, remote_name)
+    return update_localdb(localdb, user, token=None, refresh_token=None, remote=remote_name)
 
 
-def update_localdb(localdb, user, token, remote):
+def update_localdb(localdb, user, token, refresh_token, remote):
     previous_user = localdb.get_username(remote.url)
-    localdb.set_login((user, token), remote.url)
```

Comment:
renamed `set_login` to `store` and not passing a tuple but arguments. It was very confusing. Sorry for the refactor.